conbench/entities/run.py

```diff
@@ -210,6 +214,8 @@ def commit_hardware_run_map():
 class GitHubCreate(marshmallow.Schema):
     commit = marshmallow.fields.String(required=True)
     repository = marshmallow.fields.String(required=True)
+    branch = marshmallow.fields.String(required=False, allow_none=True)
+    pr_number = marshmallow.fields.String(required=False, allow_none=True)
```

Comment:
I know we don't have docs here already, but would you mind either adding them now or making an issue to do so? That would also give us the opportunity to heavily encourage folks to use pr_number only unless they really know what they are doing with the branch

Reply:
Sure! I'm all for more docs. 

Unfortunately these particular docs hide way at the bottom of the swagger page, and I'm assuming no one clicks on them. (scroll down [here](https://conbench.ursa.dev/api/docs/) and click on `BenchmarkCreate` for an example.) So that's why I didn't include them before. Plus hopefully docs for adapters/runners will detail this more. But I can add some advice here just in case!

Reply:
Yeah, that aspect of the swagger docs is decidedly not great 😞 but having soooome guidance is good and maybe once we've got some more progress we can figure out if we can make swagger better or use something else...

Reply:
They're also available inside each endpoint section if you click over from "Example value" to "Schema". And I, at least, look at these a lot when I'm trying to understand an API (n = 1)

Reply:
Oh cool!!!!!